### ISSUE TITLE
Consider laxing `max-len`

### ISSUE THREAD

**Issue by satazor** [moxystudio/eslint-config-moxy#49]
Currently, I can't have a multiline template string that contains a paragraph that has more than `140chars`:

```
        str += `
It's a bad practice having files being output with hashes during development. Your output folder will become filled with a lot of different files after a few rebuilds.

Additionally, this is know to cause ${chalk.bold('memory leaks')} with ${chalk.bold('webpack-dev-middleware\'s')} in-memory filesystem.
Either ${chalk.bold('avoid having hashes')} in development or disable in-memory filesystem through the \`memoryFs\` option.

If you feel this was a false positive, please ignore this warning.

`;
```

We should consider laxing this rules in a few cases, see https://eslint.org/docs/rules/max-len

**Comment by acostalima** [comment]
@satazor I can have a look at this as soon as we're done with https://github.com/moxystudio/eslint-config/issues/31 🙂
